navigator-menu/src/components/loop-studio-components/main/creations-container/CreationItem.jsx

```diff
@@ -0,0 +1,47 @@
+import { ItemWrapper, Backdrop, H3 } from './CreationItem.styles';
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+function CreationItem({ title, backgroundImage }) {
+  let windowWidth = window.innerWidth;
+  let backdropID = title + '-backdrop';
+
+  let design = windowWidth < 1440 ? 'mobile' : 'desktop';
+  let string = `/images/loop-studio/${design}/image-${backgroundImage}.jpg`;
+
+  useEffect(() => {
+    let wrapper = document.getElementById(title);
```

Comment:
Please use refs instead of getElementById

Actually, just use CSS for background-position and background-repeat.